aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/GetSamplingTargetsResponse.java

```diff
@@ -45,9 +45,11 @@ static SamplingTargetDocument create(
 
     abstract double getFixedRate();
 
-    abstract int getIntervalSecs();
+    @Nullable
+    abstract Integer getIntervalSecs();
```

Comment:
What is the benefit of this change? Is this just how Jackson works with optional fields?

Reply:
It defaults to "default" if not present, `0` for an int or `null` for objects. As `0` doesn't seem like a valid interval, I guess comparing to 0 instead of comparing to null would work too but it's not quite as straightforward (though does allocate a bit less memory I guess). For what it's worth, AWS SDK uses non-primitive types for all of these to indicate they could be null

Reply:
I see - and the problem *could be* that if 0 was a valid interval, we'd be treating a not-present interval the same as a provided 0-value interval.